code/upaya/app/helpers/idp_helper.rb

```diff
@@ -0,0 +1,5 @@
+module IdpHelper
+  def get_help_image_uri(question)
+    "help/#{File.basename(question['helpImageUrl'])}"
+  end
+end
```

Comment:
if this get's too big and unwieldy, sure. right now I want to add code in whatever way that gets us a working auth flow deployed to a remote server asap.